source/declarators.tex

```diff
@@ -1183,6 +1185,7 @@
 is applied to this pointer,
 the result is the pointed-to $(n-1)$-dimensional array,
 which itself is immediately converted into a pointer.
+\end{note}
```

Comment:
The alternative is to put the example inside the note, all in one paragraph. It depends on whether the example illustrates the note, or the general theme of this section.